### DISCUSSION TITLE
How do I remove the powered by footer?

### DISCUSSION THREAD

**Discussion by tmpfs** [GitbookIO/community#294] [0. General bugs and questions]
Hi,

I just upgraded to Pro as I need custom logo etc. and I thought I would be able to remove the "Powered by Gitbook" footer but I can't find it anywhere in the interface?

How do I remove the powered by Gitbook footer please?

Thanks!

**Comment by aikoii** [comment]
Hey @tmpfs! I'm afraid currently, we don't have the option to remove the GitBook watermark. However, there is a workaround you can try. You can make your space private and use our [API](https://developer.gitbook.com/) to create a branded public site 🙂.

**Reply by maxkuzkin** [reply]
Hey guys,

Love your product and it's perfectly fine to have "powered by" there, but could you place it so that it doesn't concume precious navigation bar space? Our documentation portal has a lot of elements in the left menu and that "powered by" consumes space all the time —

<img width="1448" alt="image" src="https://github.com/GitbookIO/community/assets/3930620/3b49dc4b-86ff-4658-851d-3890aa0d23ec">

Could you please consider moving it to to be the last element in navigation or the element in the footer please? As illustrated below —

<img width="1434" alt="image" src="https://github.com/GitbookIO/community/assets/3930620/bc1c499e-68db-4955-aefa-2628972b3623">

I'm 100% sure this will not harm your promo thing, but will make navigating around navigation simpler for users.
